omero/sysadmins/admins-with-restricted-privileges.txt

```diff
@@ -0,0 +1,418 @@
+allowed to do too. In case any more post-import cleaning and data
+organizing is necessary for Importer, this might be enabled by giving
+him/her also privilege necessary for Data
+organizer (see :ref:`Workflow 4` below).
+
+If you have any doubts about giving the administrators with restricted
+privilege the :term:`Sudo` privilege (which gives the ability to delete
+users' data), there are 2 workarounds which enable Importer
+to achieve import for others without :term:`Sudo`.
+
+The first workaround involves importing the data as Importer into the
+group of the future data owner and then trasferring the ownership of the
+data (see details in :ref:`Workflow 3`: Analyst). The second
+workaround involves importing into the group of the Importer as the
+importer, then moving the data into the group of the future data owner
+and then changing the ownership of the data to the owner (necessary
+tools are described in :ref:`Workflow 4`: Group and Data Organizer).
+
+Workflow 2: Importer in clients:
+
+- OMERO.importer or OMERO.insight: You have to be a member of the group
+  you want to import to in OMERO.importer or OMERO.insight. Login as the
+  administrator with restricted privileges and perform the import for
+  others as described in the chapter of Help documentation
+  :help:`import for others <facility-manager#import>`.
+
+- CLI: (see also the videos on import on
+  `OME YouTube channel
+  <https://www.youtube.com/channel/UCyySB9ZzNi8aBGYqcxSrauQ>`_)::
+
+    # Login as the Importer and sudo as the user you want to import for
+    $ bin/omero --sudo Importer -u user login
+    # Create new containers belonging to the user
+    $ bin/omero obj new Dataset name=Dataset-of-user
+    $ bin/omero obj new Project name=Project-of-user
+    # Link the containers
+    $ bin/omero obj new ProjectDatasetLink parent=Project:17 child=Dataset:13
+    # Import into created Dataset
+    $ bin/omero import ~/Desktop/CMPO1.png -T Dataset:name:Dataset-of-user
+
+.. _Workflow 3:
+
+Workflow 3: Analyst
+-------------------
+Typically, Analyst in OMERO needs to
+
+- read the data (always possible, see :ref:`Workflow 1`: Data viewer)
+- change and save the rendering settings of the images (enabled by
+  :term:`Write Data` privilege, exception is
+  Private groups, where he/she cannot save Rnd settings)
+- annotate the data (enabled by :term:`Write Data` privilege, but not
+  possible in Private groups)
+- draw and save ROIs on other users' images (enabled by :term:`Write Data`
+  privilege, but no saving in Private groups possible)
+- upload and attaching of result files to the analyzed images (enabled by
+  :term:`Write Data` privilege, except Private groups, where attaching
+  is not possible)
+- create Projects and Datasets for newly imported images in groups he/she
+  is not a member of (enabled by :term:`Write Data` privilege)
+- import new images resulting from image analysis into these containers
```

Comment:
What are "these containers"?  Projects and Datasets?  If so, why not use "Projects and Datasets"?